sdks/java/extensions/sql/zetasql/src/test/java/org/apache/beam/sdk/extensions/sql/zetasql/ZetaSQLDialectSpecTest.java

```diff
@@ -2841,6 +2842,131 @@ public void testSelectNullExceptAll() {
     pipeline.run().waitUntilFinish(Duration.standardMinutes(PIPELINE_EXECUTION_WAITTIME_MINUTES));
   }
 
+  @Test
+  public void testMultipleSelectStatementsThrowsException() {
+    String sql = "SELECT 1; SELECT 2;";
+    ZetaSQLQueryPlanner zetaSQLQueryPlanner = new ZetaSQLQueryPlanner(config);
+    thrown.expect(UnsupportedOperationException.class);
+    thrown.expectMessage(
+        "Statement list must end in a SELECT statement, and cannot contain more than one SELECT statement.");
```

Comment:
You will need to update this error message.

Reply:
Oops, fixed